strawberryfields/gbs/similarity.py

```diff
@@ -275,18 +321,11 @@ def event_cardinality(photon_number: int, max_count_per_mode: int, modes: int) -
 
 
 def prob_orbit_mc(graph: nx.Graph, orbit: list, n_mean: float = 5, samples: int = 1000) -> float:
-    """Gives a Monte Carlo estimate of the probability of a given orbit for a GBS device encoded
-    according to the input graph.
+    r"""Gives an MC estimate of the GBS probability of a given orbit according to the input graph.

```

Comment:
I prefer to spell out Monte Carlo, as it's dangerous to assume that a user will always remember what MC stands for. Personally, it makes me think of MC Hammer